src/test/groovy/org/gradle/android/RoomSchemaLocationWorkaroundTest.groovy

```diff
@@ -215,23 +215,34 @@ class RoomSchemaLocationWorkaroundTest extends AbstractTest {
 
     void assertKaptSchemaOutputsExist() {
         // Task specific schemas
-        assert file("app/build/roomSchemas/kaptDebugKotlin/org.gradle.android.example.app.AppDatabase/2.json").exists()
-        assert file("app/build/roomSchemas/kaptReleaseKotlin/org.gradle.android.example.app.AppDatabase/2.json").exists()
-        assert file("library/build/roomSchemas/kaptDebugKotlin/org.gradle.android.example.library.AppDatabase/2.json").exists()
-        assert file("library/build/roomSchemas/kaptReleaseKotlin/org.gradle.android.example.library.AppDatabase/2.json").exists()
+        assertKaptSchemaOutputsExistFor("debug")
```

Comment:
i wonder if we should add a test for flavors as well. (debug and release are build types)

`freeDebug`, `freeRelease`, `paidDebug`, `paidRelease`

At the end of the day, its the same thing though. The output directory is the same.

Reply:
Yeah, I think eventually the test application should be more realistic in general.  Right now, it has some code, resources, and a room database and some other things, but it's all very simplistic.  We've run into at least one issue where it didn't catch a problem because it was too simplistic.  I think it might be worth it at some point to create a test app that is a little more like a "real" application.